pkg/lib/bundle/types.json

```diff
@@ -175,7 +175,9 @@
       "author",
       "category",
       "rating",
-      "status"
+      "status",
```

Comment:
Adding starred and url relations to featured of Book object type, as they are used as relation blocks of bundled template